### DISCUSSION TITLE
Can someone explain show me a simple workflow for re-using existing camera/structure data to help guide results?

### DISCUSSION THREAD

**Discussion by SimonHeggie** [alicevision/Meshroom#2577] [Q&A]
Hi there, I have been scouring so many guides and websites for this one piece of illusive information. I read here and there that if there are non-estimated cameras, someone can use previously estimated camera positions to better re-calculate the scene. 

But the only way I have seen this done is via a branch of the software:

https://youtu.be/XUKu1apUuVE?si=O9SVPDRNtrxb5kHq&t=1341

So far it would seem the only way I can run this is if I use mpr's docker version of Meshroom. But I'm not in Arch Linux, I'm using Nobara which is based on Fedora, so I'm not sure if it will work.

Can anyone lend me some wisdom on this?

-S

**Comment by natowi** [comment]
If you have a fixed rig, you can run the reconstruction once and than use the sfm file as known pose data.
